packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -2,33 +2,66 @@
 
 import { Typography } from "@/src/ui/typography";
 import { useCampaigns } from "@/src/hooks/useCampaigns";
-import { useAvailableAmms } from "@/src/hooks/useAvailableAmms";
 import { Campaign, SkeletonCampaign } from "./campaign";
 
 import styles from "./styles.module.css";
```

Comment:
Here we have import after the styles.

Reply:
Addressed in e0fe2496c49b47fc15135ae02bc76bd3865e4880